lib/katello/tasks/repository.rake

```diff
@@ -59,6 +59,25 @@ namespace :katello do
     end
   end
 
+  desc "Change the download policy of all repos. Specify DOWNLOAD_POLICY=policy"
+  task :change_download_policy => ["environment", "check_ping"] do
+    policy = ENV['DOWNLOAD_POLICY']
+    unless ::Runcible::Models::YumImporter::DOWNLOAD_POLICIES.include?(policy)
+      puts "Invalid download policy specified: '#{policy}'. "
+      puts "Options are #{::Runcible::Models::YumImporter::DOWNLOAD_POLICIES.to_sentence}."
+      exit
+    end
+
+    User.current = User.anonymous_api_admin
+    repos = Katello::Repository.yum_type.where(library_instance_id: nil)
+
+    repos.find_each.with_index do |repo, index|
+      puts "Processing Repository #{index + 1}/#{repos.count}: #{repo.name} (#{repo.id})"
+      ForemanTasks.sync_task(::Actions::Katello::Repository::Update, repo,
+                             download_policy: policy)
+    end
```

Comment:
I would recommend adding a blanket rescue here to rescue and print the error with each repo but continue.

Also a test would be nice :)